packages/run-python-kit/src/nodes/run-python.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type {
+  InputValues,
+  NodeDescriberFunction,
+  NodeHandler,
+  NodeHandlerFunction,
+  Schema,
+} from "@google-labs/breadboard";
```

Comment:
It would be nice to use the new API: `@breadboard-ai/build`, but not blocking. I can help with upgrade later.